templates/index.html

```diff
@@ -67,6 +67,8 @@
   <div class="hidden" id="host-skeleton"><a class="GitHubCommitURL" href=""></a> <span class="hidden"> (<a class="GitHubDiffURL" href="">diff</a>)</span></div>
 
   <script type="text/javascript">
+  GITHUB_TOKEN = "{{.GithubToken}}";
```

Comment:
@yugui thanks for the feedback!
I agreed that avoiding sending credentials from server to client is probably not the best way. However, I cant find a better solution as of now, without forcing the user to sign in Gihub again (frontend). 

if this is okay, in your opinion, i will first merge this PR then :bow: